### PR TITLE
Switch Steam to 22.08

### PR DESCRIPTION
**Pull request by nanonyme**

### CODE REVIEW

File: com.valvesoftware.Steam.yml [hunk 1]
```diff
@@ -75,17 +75,17 @@ finish-args:
 add-extensions:
   org.freedesktop.Platform.Compat.i386:
     directory: lib/i386-linux-gnu
-    version: '22.08beta'
+    version: '22.08'
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:78]:
```suggestion
    version: *runtime-version
```

File: com.valvesoftware.Steam.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 app-id: com.valvesoftware.Steam
 runtime: org.freedesktop.Platform
-runtime-version: '22.08beta'
+runtime-version: '22.08'
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:3]:
```suggestion
runtime-version: &runtime-version '22.08'
```

File: com.valvesoftware.Steam.yml [hunk 3]
```diff
@@ -75,17 +75,17 @@ finish-args:
 add-extensions:
   org.freedesktop.Platform.Compat.i386:
     directory: lib/i386-linux-gnu
-    version: '22.08beta'
+    version: '22.08'
 
   org.freedesktop.Platform.Compat.i386.Debug:
     directory: lib/debug/lib/i386-linux-gnu
-    version: '22.08beta'
+    version: '22.08'
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:82]:
```suggestion
    version: *runtime-version
```

File: com.valvesoftware.Steam.yml [hunk 4]
```diff
@@ -96,8 +96,8 @@ add-extensions:
 
   org.freedesktop.Platform.VAAPI.Intel.i386:
     directory: lib/i386-linux-gnu/dri/intel-vaapi-driver
-    version: '22.08beta'
-    versions: '22.08beta'
+    version: '22.08'
+    versions: '22.08'
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:100]:
```suggestion
    version: *runtime-version
    versions: *runtime-version
```

File: com.valvesoftware.Steam.yml [hunk 5]
```diff
@@ -107,14 +107,14 @@ add-extensions:
   org.freedesktop.Platform.ffmpeg-full:
     directory: lib/ffmpeg
     add-ld-path: .
-    version: '22.08beta'
+    version: '22.08'
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:110]:
```suggestion
    version: *runtime-version
```

File: com.valvesoftware.Steam.yml [hunk 6]
```diff
@@ -107,14 +107,14 @@ add-extensions:
...
     no-autodownload: true
     autodelete: false
 
   org.freedesktop.Platform.ffmpeg_full.i386:
     directory: lib32/ffmpeg
     add-ld-path: .
-    version: '22.08beta'
+    version: '22.08'
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:117]:
```suggestion
    version: *runtime-version
```

File: com.valvesoftware.Steam.yml [hunk 7]
```diff
@@ -75,17 +75,17 @@ finish-args:
...
     directory: lib/debug/lib/i386-linux-gnu
-    version: '22.08beta'
+    version: '22.08'
     no-autodownload: true
 
   org.freedesktop.Platform.GL32:
     directory: lib/i386-linux-gnu/GL
     version: '1.4'
-    versions: 22.08beta;1.4
+    versions: 22.08;1.4
```
Comment (gasinvein) [inline com.valvesoftware.Steam.yml:88]:
```suggestion
    versions: &gl-versions 22.08;1.4
```

**Review by gasinvein — COMMENTED** [review]
It feels like we can replace most of the repeating branch occurrences with a yaml aliases.

**Comment by nanonyme** [comment]
bot, build

**Comment by nanonyme** [comment]
bot, build

**Pull request merged**